### ISSUE TITLE
Addon manager installing DXVK

### ISSUE THREAD

**Issue by robmart** [fmmmlee/GW2-Addon-Manager#115]
**Describe the solution you'd like**
Since SelectRender is a thing, having the option of the manager installing DXVK would be a boon towards us lazy people.  
I will admit that it wouldn't change much, however it would remove the variable of user error when using SelectRender.

**Describe alternatives you've considered**
Installing in manually? I have no idea what to write here.

**Comment by UnaSalisVictis** [comment]
agreed, especially for when dx912pxy is buggered and crashes the game after updates, this wouldbe a great addition, dxvk and dxvk-async being installed by the manager would remove user error.... i had a friend who took several tries to get it to work.
